.gitmodules

```diff
@@ -0,0 +1,8 @@
+[submodule "lib/ParallelIO"]
```

Comment:
Doesn't this depend on whether the user checks out with recursive submodules?  It may be that the code does get checked out, but it would not be built.